lib/string/strcpy/strncpytail.h

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: 2025, Alejandro Colomar <alx@kernel.org>
+// SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
> > > > Padding is a word that depends on the context.
> > > 
> > > 
> > > Could you please elaborate it?
> > > Or, preferably, put the links to some trusted sources.
> > 
> > 
> > The ISO C standard:
> > https://cigix.me/c23#6.2.6.1.p6
> > https://cigix.me/c23#6.2.6.2
> > https://cigix.me/c23#6.5.4.4.p4
> > https://cigix.me/c23#6.7.3.2.p17
> 
> These all links are about "ignored part" of something. And "ignored part" clearly have undefined value.

> How it is mapped to your term?

You should ignore the padding of null-padded character arrays.  That's why we call it padding.  And I'm not the only one doing that, FWIW.